tracer/src/Datadog.Trace/TracerManagerFactory.cs

```diff
@@ -261,7 +261,7 @@ protected virtual ITraceSampler GetSampler(TracerSettings settings)
             // Note: the order that rules are registered is important, as they are evaluated in order.
             // The first rule that matches will be used to determine the sampling rate.
 
-            if (settings.AppsecStandaloneEnabledInternal)
+            if (settings.ApmTracingEnabledInternal == false)
```

Comment:
nit
```suggestion
            if (!settings.ApmTracingEnabledInternal)
```

Reply:
I left the  `== false` for reading clarity